### PR TITLE
Feature: Initialize the files for the project

### PR DESCRIPTION
**Pull request by momondad-demo**
[Pull Request message]
Write what is in this pull request. As clear as possible for reviewers easily to get it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+# Vue Crash Course (TodoList)
+
+This is the code for [the crash course on YouTube](https://youtu.be/Wy9q22isx3U)
+
+## Quick Start
+
+```bash
+# Install dependencies
+npm install
+
```
Comment (k-dovan) [inline README.md:10]: Great man, xD

File: package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  },
+  "devDependencies": {
+    "@vue/cli-plugin-babel": "^3.3.0",
+    "@vue/cli-plugin-eslint": "^3.3.0",
+    "@vue/cli-service": "^3.3.0",
+    "babel-eslint": "^10.0.1",
+    "eslint": "^5.8.0",
+    "eslint-plugin-vue": "^5.0.0",
+    "vue-template-compiler": "^2.5.21"
+  },
```
Comment (k-dovan) [inline package.json:24]: Well done xD

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title>vue_crash_todolist</title>
+  </head>
```
Comment (k-dovan) [inline public/index.html]: Fix this now ;(

**Review by k-dovan — CHANGES_REQUESTED** [review]
This is suck

**Review by k-dovan — APPROVED** [review]
That's good man 🗡️

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title>vue_crash_todolist project title</title>
```
Comment (momondad-demo) [inline public/index.html]:
```suggestion
    <title>vue_crash_todolist project title hahaha just like this!!!!</title>
```

Comment (k-dovan) [inline public/index.html]:
```suggestion
    <title>Learning Vuejs from scrash!!!</title>
```

**Review by k-dovan — CHANGES_REQUESTED** [review]

**Review by momondad-demo — COMMENTED** [review]
what is it? man xd

**Review by k-dovan — APPROVED** [review]
Easy, right?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    ],
+    "rules": {},
+    "parserOptions": {
+      "parser": "babel-eslint"
+    }
+  },
+  "postcss": {
+    "plugins": {
+      "autoprefixer": {}
+    }
```
Comment (momondad-demo) [inline package.json:42]: Poor myself ;((( Me do it now. Me is me 🗡️

File: README.md [hunk 6]
```diff
@@ -1,2 +1,16 @@
...
+# Serve on localhost:8080
+npm run serve
+
+# Build for production
+npm run build
+```
```
Comment (k-dovan) [inline README.md:16]: what is the three dots? WTH xD

File: src/components/AddTodo.vue [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+      this.title = '';
+    }
+  }
+}
+</script>
+
+<style scoped>
+  form {
+    display: flex;
+  }
```
Comment (k-dovan) [inline src/components/AddTodo.vue:37]: This 2

**Review by k-dovan — CHANGES_REQUESTED** [review]
Make some changes now xD

**Review by k-dovan — CHANGES_REQUESTED** [review]
Next one

File: src/components/AddTodo.vue [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    </form>
+  </div>
+</template>
+
+<script>
+// import uuid from 'uuid';
+export default {
+  name: "AddTodo",
+  data() {
+    return {
```
Comment (k-dovan) [inline src/components/AddTodo.vue:15]: This 4

File: src/components/AddTodo.vue [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    addTodo() {
+      const newTodo = {
+        title: this.title,
+        completed: false
+      }
+      // Send up to parent
+      this.$emit('add-todo', newTodo);
+
+      this.title = '';
+    }
```
Comment (k-dovan) [inline src/components/AddTodo.vue:29]: This 5

File: src/components/AddTodo.vue [hunk 10]
```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <form @submit.prevent="addTodo">
+      <input type="text" v-model="title" name="title" placeholder="Add Todo...">
+      <input type="submit" value="Submit" class="btn">
+    </form>
```
Comment (k-dovan) [inline src/components/AddTodo.vue:6]: and much more Xd

File: public/index.html [hunk 11]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title>Learning Vuejs from scrash!!!</title>
+  </head>
```
Comment (k-dovan) [inline public/index.html:9]: I can not stand with this any more xD

**Review by k-dovan — CHANGES_REQUESTED** [review]
Do these things right now xD

File: README.md [hunk 12]
```diff
@@ -1,2 +1,16 @@
-# PullRequests
- 
+# Vue Crash Course (TodoList)
```
Comment (k-dovan) [inline README.md]:
```suggestion
# Vue Crash Course (TodoList)!!!
```

**Review by k-dovan — CHANGES_REQUESTED** [review]

**Review by k-dovan — APPROVED** [review]

**Comment by momondad-demo** [comment]
Initialize the files for the project. Well done!

**Pull request merged**